### ISSUE TITLE
use local admin account while AD (LDAP) is active & logs

### ISSUE THREAD

**Issue by Shaddowrunner414** [ansible-semaphore/semaphore#1550]
Hi there, 
we were setting up Semaphore and really like it, but we have a problem (and a half). 
We want to use our Active Directory but in order to assign users to roles, we need the local admin account, right?
But when we enable LDAP and try to log in, it goes straight to the LDAP and gives a "invalid credentials" error no matter if we use the credentials of the local admin account or valid credentials from AD.

Is it even possible to use the local admin account in parallel with LDAP? If not, how to add users?

Is our assumption correct, that once AD is connected correctly, there should be all the users from AD in Semaphore > Team > New Team Member > User? Right now there is "no data available", but AD users could be assigned roles in Semaphore to be able to log in to semaphore using the AD account?

And in order to debug this, we wanted to turn up the trace-level, but could not figure out how to do it. All we have is the docker logs as we run semaphore in a docker container, and they are not very detailed. How can we access the semaphore logs?

**Comment by DevAlphaKilo** [comment]
@Shaddowrunner414 
What version are you using? There is a known issue (#1522) in the current latest release. The functionality you desire is working currently in **v2.9.4**, just confirmed it by using my **local_admin** account after signing out of my LDAP account. If possible, try again using **v2.9.4** to see if downgrading fixes the issue.

**Comment by Shaddowrunner414** [comment]
we indeed have 2.9.37 installed (don't know how I didn't notice your ticket when searching for a fix). I'll report back after downgrading

**Comment by Shaddowrunner414** [comment]
We downgraded to v2.9.4 and tried again, but basically the same result. So I fall back to my second question: How can we access the semaphore logs?

**Comment by EHEX-schildt** [comment]
> Is our assumption correct, that once AD is connected correctly, there should be all the users from AD in Semaphore > Team > New Team Member > User? Right now there is "no data available", but AD users could be assigned roles in Semaphore to be able to log in to semaphore using the AD account?

No. A user needs to login once to get created in the database, after that first login you can assigned roles/teams
#1139 

> Is it even possible to use the local admin account in parallel with LDAP? If not, how to add users?

works for me in v2.9.4 we link users via api as local admin

**Comment by Bodzz96** [comment]
I think I have the same issue on v2.11.2, when LDAP is configured 'm not able to login with local admin or domain users with error wrong creds for local admin

**Comment by muelli01** [comment]
I recently installed 2.17.26 on two different servers, same installation procedure. Only difference is, the servers are in a different domain using different AD servers. The servers clearly differ in behavior. on one sever i can login using the local admin account with AD enabled. On the other server the local admin account is sent to AD and fails. I removed the DB, reinstalled, no matter what I do, on that server I cannot get the local admin to work.

edit: I tried 2.18.4, same

edit 2: I turned on ldap debugging, turns out the server returned a data 775 which was a hint. seems there was a (disabled) user in the AD with the same name as the local admin. Thus, the problem is solved for me :)
